basis/ui/backend/gtk/gtk.factor

```diff
@@ -420,9 +425,11 @@ M: window-handle flush-gl-context ( handle -- )
 
 M:: gtk-ui-backend (open-window) ( world -- )
     GTK_WINDOW_TOPLEVEL gtk_window_new :> win
+    gtk_drawing_area_new :> drawable
+    win drawable gtk_container_add
```

Comment:
In particular, if any opengl calls happen during the configure signal on the window, gtk (on unity) will send the events for all the frames. But if you do your opengl calls during the configure on the drawable, gtk will coalesce the events. At least that's what I saw empirically, didn't find docs for that.